setup.py

```diff
@@ -9,14 +9,179 @@
+                     '        def before(self):\n'
+                     '            self.start = time.time()\n'
+                     '\n'
+                     '        def after(self):\n'
+                     '            duration = time.time() - self.start\n'
+                     "            treshold = self.params.get('threshold')\n"
+                     '            if threshold and duration > threshold:\n'
+                     "                raise Exception('Execution took longer "
+                     "than the threshold')\n"
+                     '\n'
+                     '\n'
+                     '    @threshold(threshold=2)\n'
+                     '    def my_function():\n'
+                     '        time.sleep(3)\n'
+                     '\n'
+                     '    >>> my_function()\n'
+                     '    Exception: Execution took longer than the threshold\n'
+                     '\n'
+                     '\n'
+                     'Async Decorator\n'
+                     '---------------\n'
+                     '\n'
+                     "There's also support for decorating coroutines (or any "
+                     'awaitable), for Python >=3.5 only.\n'
+                     '\n'
+                     'The implementation is the same as with the sync version, '
+                     'just inherit from\n'
+                     ':class:`~fqn_decorators.async.AsyncDecorator` instead.\n'
+                     '\n'
+                     'Example::\n'
+                     '\n'
+                     '    import asyncio\n'
+                     '    import time\n'
+                     '    from fqn_decorators.async import AsyncDecorator\n'
+                     '\n'
+                     '    class time_it_async(AsyncDecorator):\n'
+                     '\n'
+                     '        def before(self):\n'
+                     '            self.start = time.time()\n'
+                     '\n'
+                     '        def after(self):\n'
+                     '            duration = time.time() - self.start\n'
+                     '            print("{0} took {1} '
+                     'seconds".format(self.fqn, duration))\n'
+                     '\n'
+                     '    @time_it_async\n'
+                     '    async def coro():\n'
+                     '        await asyncio.sleep(1)\n'
+                     '\n'
+                     '    >>> loop = asyncio.get_event_loop()\n'
+                     '    >>> loop.run_until_complete(coro())\n'
+                     '    __main__.coro took 1.001493215560913 seconds\n'
+                     '\n'
+                     '\n'
+                     '.. end_usage\n',
  'name': 'fqn-decorators',
  'packages': ['fqn_decorators'],
  'tests_require': ['tox'],
- 'url': 'ssh://git@github.com:kpn-digital/py-fqn-decorators.git',
- 'version': '1.0.0',
+ 'url': 'https://github.com/kpn-digital/py-fqn-decorators',
+ 'version': '1.1.0+1.gd44f626',
```

Comment:
Is this version the recommended way of doing it? I would expect the committed setup.py to have a more specific version.